flows/actions/testdata/call_webhook.json

```diff
@@ -469,5 +469,72 @@
                 }
             ]
         }
+    },
+    {
+        "description": "Error event when body exceeds download limit",
+        "http_mocks": {
+            "http://temba.io/": [
+                {
+                    "status": 200,
+                    "body": "X",
+                    "body_repeat": 100010
+                }
+            ]
+        },
+        "action": {
+            "type": "call_webhook",
+            "uuid": "ad154980-7bf7-4ab8-8728-545fd6378912",
+            "method": "GET",
+            "url": "http://temba.io/",
+            "result_name": "My Webhook"
+        },
+        "events": [
+            {
+                "created_on": "2018-10-18T14:20:30.000123456Z",
+                "step_uuid": "59d74b86-3e2f-4a93-aece-b05d2fdcde0c",
+                "text": "webhook response body exceeds 100000 bytes limit",
+                "type": "error"
+            },
+            {
+                "created_on": "2018-10-18T14:20:30.000123456Z",
+                "elapsed_ms": 0,
+                "request": "GET / HTTP/1.1\r\nHost: temba.io\r\nUser-Agent: goflow-testing\r\nAccept-Encoding: gzip\r\n\r\n",
+                "response": "HTTP/1.0 200 OK\r\nContent-Length: 100010\r\n\r\n",
+                "status": "connection_error",
```

Comment:
should this be a different status.. like `download_error`? These statuses are only used for event logging